File: lua/refactoring/tests/refactor/123/cpp/multiple-vars/inline_var.expected.cpp
#include <algorithm>

struct Order {
  int quantity;
  int itemPrice;
};

float orderCalculation(Order order) {
  auto i = 3;

  return order.quantity * order.itemPrice -
         std::max(0, order.quantity - 500) * order.itemPrice * 0.05 +
         std::min(order.quantity * order.itemPrice * 0.1, 100.0);
}

File: lua/refactoring/tests/refactor/123/lua/multiple-vars/inline_var.expected.lua
-- stylua: ignore start

local function orderCalculation(order)
    local i = 3

    return order.quantity * order.itemPrice
        - math.max(0, order.quantity - 500) * order.itemPrice * 0.05
        + math.min(order.quantity * order.itemPrice * 0.1, 100)
end
